protocols/tests/test_migration/test_migration_reports_4_0_0_to_reports_4_2_0.py

```diff
@@ -4,6 +4,8 @@
 from protocols import reports_4_2_0
 from protocols.migration import MigrateReports400To420
 from protocols.util import get_valid_cancer_interpretation_request_4_0_0
+from protocols.util.dependency_manager import VERSION_310, VERSION_400
```

Comment:
Can we please change this line to:
```
from protocols.util.dependency_manager import VERSION_400
from protocols.util.dependency_manager import VERSION_300
```
or 
```
from protocols.util import dependency_manager
```
and refer to `dependency_manager.VERSION_400` and `dependency_manager.VERSION_300`